### ISSUE TITLE
Running Exit on USDT perpetual websocket Error

### ISSUE THREAD

**Issue by XhrCkYsSgZokJL** [bybit-exchange/pybit#28]
Code: 
```
client = usdt_perpetual.WebSocket(test=False, trace_logging=False)   
client.exit()
```

Output:
```'WebSocket' object has no attribute 'ws'```

**Comment by dextertd** [comment]
That'll be because `_connect()` hasn't been called yet so there's no websocket connection open yet. How best to handle it? Catch it and then raise that a connection hasn't been opened yet? I'm not sure

**Comment by dextertd** [comment]
Related to this issue, and I think it would be solved by this issue's enhancement being implemented #87
